src/runtime.js

```diff
@@ -710,7 +710,10 @@ class MultidocManager {
             const body = importShadowBody(
                 shadowRoot, doc.body, /* deep */ true);
             body.classList.add('amp-shadow');
-            shadowRoot.appendChild(body);
+            // TODO (torch2424): Understand why the body is duplicated, when we are creating the shadow root element
```

Comment:
That should be `DO NOT SUBMIT` until we resolve. By API body attachment must be here. This is important because there are numerous processes kicked of in startup as soon as body is appended, so we need to know when it happens pretty well.